code/game/objects/items/grenades/plastic.dm

```diff
@@ -141,6 +141,26 @@
 			message_say = "FOR ME MATEYS!"
 		else if(our_guy.has_antag_datum(/datum/antagonist/wizard))
 			message_say = "FOR THE FEDERATION!"
+		else if(our_guy.has_antag_datum(/datum/antagonist/heretic))
+			message_say = "MANSUS SMILES UPON ME!"
+		else if(our_guy.has_antag_datum(/datum/antagonist/abductor))
+			message_say = "FOR THE MOTHERSHIP!"
+		else if(our_guy.has_antag_datum(/datum/antagonist/brainwashed))
+			message_say = "FOR-" //Ambiguity is probably best for brainwashed people
+		else if(our_guy.has_antag_datum(/datum/antagonist/obsessed) || our_guy.has_antag_datum(/datum/antagonist/valentine))
+			message_say = "FOR MY LOVE!"
+		else if(our_guy.has_antag_datum(/datum/antagonist/monkey))
+			message_say = "FOR THE CONSORTIUM!"
+		else if(our_guy.has_antag_datum(/datum/antagonist/nightmare))
+			message_say = "FOR THE DARKNESS!"
+		else if(our_guy.has_antag_datum(/datum/antagonist/santa))
+			message_say = "FOR CHRISTMAS!" //The sheer absurdity of this
+		else if(our_guy.has_antag_datum(/datum/antagonist/separatist))
+			message_say = "FOR THE MOTHERLAND!"
+		else if(our_guy.has_antag_datum(/datum/antagonist/survivalist) || our_guy.has_antag_datum(/datum/antagonist/heartbreaker))
+			message_say = "FOR ME!"
+		else if(our_guy.has_antag_datum(/datum/antagonist/wishgranter))
```

Comment:
this `if else` chain is now twenty one checks long

can we just make antagonistic declaration message a variable on /datum/antagonist?

Reply:
Yes this is what I was thinking

Reply:
Probably a smart optimization.  Going to ~~hit byond with a hammer~~ attempt.